sources/platform/proxy/index.md

```diff
@@ -94,40 +100,3 @@ if __name__ == '__main__':
     />
 </CardGrid>
 
-**For pricing information, visit [apify.com/proxy](https://apify.com/proxy).**
```

Comment:
I moved this to "Usage" as it belongs there. We could list it here as features but I don't think it's necessary for the main landing page with explanation